lib/src/products_page.dart

```diff
@@ -41,7 +47,39 @@ class _Products extends StatelessWidget {
 class _Product extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Text('Product Item');
+    return Card(
+      child: Column(
+        children: <Widget>[
```

Comment:
Let's apply these props to the Column Widget so that it expands to fill the available space:
`mainAxisSize: MainAxisSize.max,
        mainAxisAlignment: MainAxisAlignment.spaceBetween,`

Reply:
Done with commit  26ccfb62d95c765ae078e7a068c78eeeb9c23457